source/Changelog.txt

```diff
@@ -1,6 +1,10 @@
 13/05/2024 - Dragon Slayer
 	Added New Shield Type 107 so shield ID's no longer have to be in hard code for shields to work properly
 
+11/05/2024 - Dragon Slayer/Xuri/Maarc
+	Added Support for new AOS property tag.
+	-DURABILITYHPBONUS - Add durability (aka Health) to the hp and max hp as a bonus
```

Comment:
Just DURABILITYHPBONUS as a standalone tag, or with a value? And if a value, does it represent additional hitpoints or a percentage multiplier?